### ISSUE TITLE
MathJax equations are wide or not inline when using MathJax locally in ver 3.0.5

### ISSUE THREAD

**Issue by ReductoAdAbsurdum** [mathjax/MathJax#2438]
### Issue Summary

the equations are a bit corrupted...they are a bit above the line and the plus sign is not rendered well when using **tex-svg.js** ..... The spacing in equations are off when using **tex-chtml.js**.

### Steps to Reproduce:

This is what I get when I use the following code in my html: **(mathjax ver 2)**

`<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>`

https://ibb.co/6J8LQbs

This is what I get when I use mathjax locally:**(mathjax ver 3)**

`<script id="MathJax-script" async src="MathJax/es5/tex-chtml.js"></script>`

https://ibb.co/N1Cz6Qc

`<script id="MathJax-script" async src="MathJax/es5/tex-svg.js"></script>`

https://ibb.co/6Wxgts2

As you can see, when i use MathJax locally, the equations are a bit corrupted...they are a bit above the line and the plus sign is not rendered well .....

Is there a way to fix this ? 

### Technical details:

* MathJax Version: 3.0.5
* Client OS: Windows 10 pro
* Browser: Google Chrome is up to date...Version 81.0.4044.138 (Official Build) (64-bit)

**Comment by dpvc** [comment]
I suspect that there is CSS on your page that is interfering with MathJax's output.  I can't check on that without a link to a page that exhibits the issue, but that is certainly what it looks like.

You could try (temporarily) disabling your CSS files and see if the equations render properly.  If so, then put back your CSS one file at a time until you find the one that is causing the problems, then look through that too see what might be causing the trouble.  Often it is a `*` to apply to all elements.

Alternatively, you can use the browser's DOM inspector to select the individual elements in the MathJax output and see if any of your CSS rules are overriding the ones from MathJax.

**Comment by ReductoAdAbsurdum** [comment]
> I suspect that there is CSS on your page that is interfering with MathJax's output. I can't check on that without a link to a page that exhibits the issue, but that is certainly what it looks like.
> 
> You could try (temporarily) disabling your CSS files and see if the equations render properly. If so, then put back your CSS one file at a time until you find the one that is causing the problems, then look through that too see what might be causing the trouble. Often it is a `*` to apply to all elements.
> 
> Alternatively, you can use the browser's DOM inspector to select the individual elements in the MathJax output and see if any of your CSS rules are overriding the ones from MathJax.

That was it...
the issue was this line: 

`*{   margin: 2px; padding: 1px; }`

replaced * with tag names and the issue is gone.

**Comment by dpvc** [comment]
Glad you were able to track it down and resolve it!
